packages/loader/container-loader/src/container.ts

```diff
@@ -767,7 +781,7 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
             this._attachState = AttachState.Attached;
             this.emit("attached");
             this.cachedAttachSummary = undefined;
-
+            this.maxClientLeaveWaitTime = request.headers?.[LoaderHeader.maxClientLeaveWaitTime];
```

Comment:
this will just go away when we reuse the config

Reply:
Removed.